pwnlib/util/__init__.py

```diff
@@ -1,19 +1,5 @@
 from __future__ import absolute_import
 
-from pwnlib.util import crc
```

Comment:
It seems that this will break `import pwnlib.util` and then doing e.g. `pwnlib.util.cyclic.xyz`, no?

Reply:
Yes, it will. But!

But this removal saves us partially from the hell of circular imports. Like when `pwnlib.xyz` uses `pwnlib.util.misc`, but `pwnlib.util.fiddling` uses `pwnlib.xyz`, this is not a circular import normally, but all those imports here make it such if `from . import fiddling` is before `from . import misc`. And this sometimes depends on the order of imports elsewhere. So I decided that we can sacrifice convenience in favor of flexibility here.